src/AT_RDD_ShellAveraged.c

```diff
@@ -41,22 +43,49 @@
   return 2.0 * Katz_point_coeff_Gy * ( log(r2_m/r1_m) - (r2_m - r1_m)/max_electron_range_m ) / (gsl_pow_2(r2_m/max_electron_range_m) - gsl_pow_2(r1_m/max_electron_range_m));
 }
 
+double   beta0(double x, double a, double iteration_threshold) {
+    if (x < 0.99) {
+        // This iterative algorithms computes the value of the beta function very accurately
+        // but converges very slowly when the value of x is very close to 1.
+        double ret = 0.0;
+        double xi = 1.0;
+        long i = 0;
+        while (xi > iteration_threshold) {
+            ret += xi / (a + i);
+            xi *= x;
+            i += 1;
+        }
+        ret *= pow(x, a);
+        return ret;
+    }
+    else {
+        // Series expansion taken from https://www.wolframalpha.com/input/?i=int+-1%2Fx+%281+-+x%29%5E%28a-1%29+dx
```

Comment:
Wolfram alpha provides series expansion at x = 0. Here we need series expansion for x = 1.
Explain in comment how did you get from one case to another one.

Reply:
I've added an explanation in the new commit.